controllers/kafkatopic_controller.go

```diff
@@ -32,10 +35,18 @@ import (
 	"github.com/banzaicloud/koperator/pkg/k8sutil"
 	"github.com/banzaicloud/koperator/pkg/kafkaclient"
 	"github.com/banzaicloud/koperator/pkg/util"
+	"github.com/banzaicloud/koperator/pkg/webhooks"
 )
 
 var topicFinalizer = "finalizer.kafkatopics.kafka.banzaicloud.io"
 
+func isTopicManagedByKoperator(topic metav1.Object) bool {
+	if manager, ok := topic.GetAnnotations()[webhooks.TopicManagedByAnnotationKey]; !ok || strings.ToLower(manager) == webhooks.TopicManagedByKoperatorAnnotationValue {
```

Comment:
**Recommended**: We **SHOULD NOT** call the annotation `manager` IMO because it is misleading.

```suggestion
	if managedByAnnotation, ok := topic.GetAnnotations()[webhooks.TopicManagedByAnnotationKey]; !ok || strings.ToLower(managedByAnnotation) == webhooks.TopicManagedByKoperatorAnnotationValue {
```